app/common/app_config.js

```diff
@@ -1,7 +1,7 @@
 "use strict";
 
 const Store = require('electron-store');
-
+const { isDevEnv } = require('./utils');
```

Comment:
Style detail, while this way is generally accepted as a good syntax for requires, I'm using a more basic syntax in the rest of the project (partially for personal preference to be honest), just for consistency. Could you change this for a syntax according to the rest of the code?
Something like this
```js
const utils=require('./utils');
// Rest of the code
utils.isDevEnv()
```

Thanks

Reply:
Yup, no problemo!